### PR TITLE
Various fixes and some colour work

### PR DESCRIPTION
**Pull request by EllipticEllipsis**
- Fix game summaries to read data properly
- Progress chart and material colours, taken from a couple of Highcharts presets
- Add a theme (which isn't used properly because Angular)
- Fix statement button colour, box colours, `ul`/`li` styling scope
- Stick header to the top
- Fix chart tooltip

### CODE REVIEW

File: src/app/games/game/game-progress/game-chart/game-chart.component.ts [hunk 1]
```diff
@@ -2,6 +2,12 @@ import { Component, Input, OnChanges, OnInit } from '@angular/core';
 import { Chart } from 'angular-highcharts';
 import { Options, PointOptionsObject } from 'highcharts';
 import { IChart } from 'src/app/games/games.service.interface';
+import * as Highcharts from 'highcharts';
+
+import theme from 'highcharts/themes/avocado';
+// import theme from 'highcharts/themes/sand-signika';
+// import theme from 'highcharts/themes/dark-blue';
```
Comment (rozzan) [inline src/app/games/game/game-progress/game-chart/game-chart.component.ts]: Please remove these :)

File: src/app/games/game/game-progress/game-chart/game-chart.component.ts [hunk 2]
```diff
@@ -50,17 +56,22 @@ export class GameChartComponent implements OnChanges {
...
       max = null;
     }
 
     const options: Options = {
-      chart: { type: "line" },
+      chart: {
+        type: "line",
+        // backgroundColor: 'rgba(0, 0, 0, 0.5)'
+        // backgroundColor: 'rgba(0, 0, 0, 0.0)'
+        // backgroundColor: "#efefef"
```
Comment (rozzan) [inline src/app/games/game/game-progress/game-chart/game-chart.component.ts]: Remove these also please :)

File: src/app/games/game/game-progress/game-progress.component.scss [hunk 3]
```diff
@@ -9,12 +9,18 @@ hr {
   float: left;
 }
 
-.matching {
-  color: #01ce47;
+.nonmatching {
+  // color: #ffc107;
+  // color: #F3E796; // Avocado
+  // color: #f45b5b; // Sand Signika
```
Comment (rozzan) [inline src/app/games/game/game-progress/game-progress.component.scss]: Removeo pleaseo :)

File: src/app/games/game/game-progress/game-progress.component.scss [hunk 4]
```diff
@@ -9,12 +9,18 @@ hr {
...
+  // color: #f45b5b; // Sand Signika 
+  color: #DDDF0D; // 
 }
 
-.nonmatching {
-  color: #ffc107;
+.matching {
+  // color: #01ce47;
+  // color: #95C471; // Avocado
+  // color: #8085e9; // Sand Signika
```
Comment (rozzan) [inline src/app/games/game/game-progress/game-progress.component.scss]: Removeu pleaseu :)

File: src/styles.scss [hunk 5]
```diff
@@ -32,10 +64,20 @@ markdown h2 {
...
+
+mat-dialog-content {
+  ul {
+    padding-left: 20px;
+  }
+  
+  li {
+    margin-bottom: 1rem;
+  }
+}
```
Comment (rozzan) [inline src/styles.scss]: This can be moved to the dialog component's SCSS file.

**Review by rozzan — COMMENTED** [review]
Just comment removal and refactoring to be done :)

File: src/styles.scss (hunk 5 above)
Comment (EllipticEllipsis) [reply src/styles.scss]: I tried putting them in dialog.component.scss, but it didn't work for some reason.

Comment (EllipticEllipsis) [reply src/styles.scss]: And without the mat-dialog-content wrapper, same thing.

Comment (rozzan) [reply src/styles.scss]: Can use `::ng-deep` to make the rule propagate to the content within `innerHTML`. ex. `::ng-deep ul { ... }`

Comment (EllipticEllipsis) [reply src/styles.scss]: Oh, there we go.

**Comment by EllipticEllipsis** [comment]
Pushed shadow and menu colours.

**Pull request merged**